src/pretix/presale/templates/pretixpresale/event/fragment_cart.html

```diff
@@ -392,12 +395,20 @@
         {% endfor %}
     {% endfor %}
     {% for fee in cart.fees %}
-        <div class="row cart-row">
-            <div class="col-md-4 col-xs-6">
+        <div role="row" class="row cart-row">
+            <div role="cell" class="product">
                 <strong>{{ fee.get_fee_type_display }}</strong>
             </div>
+            {% if download %}
+                <div role="cell" class="download-desktop"></div>
+            {% else %}
+                <div role="cell" class="count"></div>
+                {% if not hide_prices %}
+                    <div role="cell" class="singleprice price"></div>
+                {% endif %}
+            {% endif %}
```

Comment:
This creates useless visual spacing on mobile.

Before (fee title and price on the same line):

![image](https://github.com/user-attachments/assets/4d5459ec-e4da-4adb-a37b-bf3c971d186f)


After (price one line down below):

![image](https://github.com/user-attachments/assets/05702e44-fbd7-4e38-af9a-747cb59f4893)

Should this maybe be sr-only?

Reply:
How could I not have seen this!? Fixed.